src/components/Main/Banner/ClubPromotion.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import styles from '../../styles/Main/ClubPromotion.module.scss';
+import styles from '../../../styles/Main/ClubPromotion.module.scss';
```

Comment:
```suggestion
import styles from 'styles/Main/ClubPromotion.module.scss';
```
styles 경로 쓸때 ../ 다 지우고 styles 부터 시작해주세여 나중에 파일 위치 바꿀때 에러남